certified-connectors/MicrosoftGraphSecurity/apidefinition.swagger.json

```diff
@@ -350,7 +346,7 @@
               "properties": {
                 "resource": {
                   "type": "string",
-                  "description": "Specify the resource that will be monitored for changes. Do not include base URL (https://graph.microsoft.com/v1.0/). Include security/alerts followed by the odata query. For e.g. security/alerts?$filter=status eq �New�",
+                  "description": "Specify the resource that will be monitored for changes. Do not include base URL (https://graph.microsoft.com/v1.0/). Include security/alerts followed by the odata query. For e.g. security/alerts?$filter=status eq �New�",
```

Comment:
Why is this Unicode character used here?

Reply:
oh weird, that was a VS Code file formatting/saving thing. I think these are bad UTF-8 chars.